### PR TITLE
Add 2019-11-17 ~ 2019-11-22 weekly update

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

File: content/2019/2019-11/2019-11-22-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    
+    https://github.com/apache/pulsar/wiki/PIP-38%3A-Batch-Receiving-Message
+
+### Notable Feature
+
+- [Connector] Support CDC connector for MongoDB (Release: 2.5.0).
+
+    https://github.com/apache/pulsar/pull/5590
+    
+- [Functions] Parametrize grpc and metrics ports so that they can be configured (Release: 2.5.0).
```
Comment (Jennifer88huang) [inline content/2019/2019-11/2019-11-22-pulsar-weekly.md]: "Parametrize": is this correct spelling?

File: content/2019/2019-11/2019-11-22-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    https://github.com/apache/pulsar/pull/5023
+    
+- [Client][Python] Fix Python consumer receive always timeout after 100ms (Release: 2.4.3, 2.5.0).
+
+    https://github.com/apache/pulsar/pull/5706
+
+### Event/News
+
+* [Conference] KubeCon + CloudNativeCon North America 2019 was held on November 18-21 in San Diego. Yahoo! Japan was one of the gold sponsors and Nozomi Kurihara presented and promoted Pulsar at this conference.
```
Comment (Jennifer88huang) [inline content/2019/2019-11/2019-11-22-pulsar-weekly.md]: 1. Yahoo! JAPAN
2. Pulsar team and Athenz team from Yahoo!JAPAN attended the conference as booth exhibitor.

File: content/2019/2019-11/2019-11-22-pulsar-weekly.md (hunk 1 above)
Comment (Anonymitaet) [reply content/2019/2019-11/2019-11-22-pulsar-weekly.md]: correct

File: content/2019/2019-11/2019-11-22-pulsar-weekly.md (hunk 2 above)
Comment (Anonymitaet) [reply content/2019/2019-11/2019-11-22-pulsar-weekly.md]: Thank you

**Comment by Anonymitaet** [comment]
@Jennifer88huang PTAL

**Pull request merged**